ledger/ledger.go

```diff
@@ -486,13 +499,22 @@ func (l *Ledger) LookupAccount(round basics.Round, addr basics.Address) (data le
 	l.trackerMu.RLock()
 	defer l.trackerMu.RUnlock()
 
-	data, rnd, rewardsVersion, rewardsLevel, err := l.accts.lookupWithoutRewards(round, addr, true /* take lock */)
+	baseData, rnd, rewardsVersion, rewardsLevel, err := l.accts.lookupWithoutRewards(round, addr, true /* take lock */)
 	if err != nil {
 		return ledgercore.AccountData{}, basics.Round(0), basics.MicroAlgos{}, err
 	}
 
 	// Intentionally apply (pending) rewards up to rnd, remembering the old value
 	withoutRewards = data.MicroAlgos
+
+	// mixin online data
+	onlineData, err := l.acctsOnline.lookupOnlineAccountData(rnd, addr)
```

Comment:
like we discussed yesterday, we can keep the voting data in the regular AccountData / acct updates tracker